src/i18n/locales/en/translation.json

```diff
@@ -465,8 +465,8 @@
           }
         },
         "remove": {
-          "title": "Remove This Account",
-          "subtitle": "Removes all allowances, payment data and keys associated with this account.",
+          "title": "Disconnect This Wallet",
+          "subtitle": "All the linked allowances will be deleted. Please be certain.",
```

Comment:
Change to: "Removes all allowances, payment data and keys associated with this wallet."